common/src/abstractions/api.service.ts

```diff
@@ -379,6 +380,7 @@ export abstract class ApiService {
     postOrganizationApiKey: (id: string, request: PasswordVerificationRequest) => Promise<ApiKeyResponse>;
     postOrganizationRotateApiKey: (id: string, request: PasswordVerificationRequest) => Promise<ApiKeyResponse>;
     postOrganizationUpgrade: (id: string, request: OrganizationUpgradeRequest) => Promise<PaymentResponse>;
+    postOrganizationUpdateSubscription: (id: string, request: OrganizationSubscriptionUpdateRequest) => Promise<void>;
```

Comment:
Any particular reason for using `void` here? I think we usually use `any` in the `ApiService`.

Reply:
The reason is better type safety. 

Requests that do not have a return value should return void to indicate that fact. The way the `ApiService.send` method works is it returns a response only if `hasResponse == true`, otherwise it will either return `void` or `throw`. This way, no component using this api method could mistakenly think it's getting a response object.